aws/dns/stages.tf

```diff
@@ -0,0 +1,22 @@
+resource "aws_route53_zone" "dns_zone" {
```

Comment:
Why is this file called `stages.tf`?

Reply:
the file will be copied into the stages (prod, staging, dev) in `Dockerfiles`.
the file is re-used.
not sure what to call it better.

Reply:
maybe just `dns`?

Reply:
Ok, I think perhaps we are overloading `dns/`. 

Thinking we should have `root-dns` and maybe `account-dns`

Reply:
(open to alternative names too... but i think we should create more folders rather than less)

Reply:
A given folder should never serve 2 purposes.